core/src/main/java/com/linecorp/armeria/common/metric/MoreMeterBinders.java

```diff
@@ -41,8 +42,11 @@ public final class MoreMeterBinders {
 
     static {
         // Bind the default Netty allocator metrics to the default MeterRegistry.
-        new NettyAllocatorMetrics(PooledByteBufAllocator.DEFAULT)
-                .bindTo(Flags.meterRegistry());
+        final ByteBufAllocator alloc = ByteBufAllocator.DEFAULT;
```

Comment:
`PooledByteBufAllocator` is no longer the default allocator. https://github.com/netty/netty/wiki/Netty-4.2-Migration-Guide#:~:text=Explicitly%20configure%20your,allocator.type%3Dpooled.